spec/models/solidus_pay_tomorrow/gateway_spec.rb

```diff
@@ -41,15 +41,38 @@
     end
   end
 
-  describe '#authorize' do
-    it 'raises NotImplementedError' do
-      expect { described_class.new.authorize }.to raise_error(NotImplementedError)
+  describe '#void' do
+    let(:void_success_response) do
+      { status: 'ok',
+        message: 'application cancelled',
+        token: nil,
+        maxApprovalAmount: nil,
+        lender: nil }.stringify_keys!
+    end
+
+    it 'returns an active merchant billing success response' do
+      allow(SolidusPayTomorrow::Client::VoidService).to receive(:call).with(
+        order_token: payment.response_code,
+        payment_method: payment_method
+      ).and_return(void_success_response)
+      result = described_class.new.void(payment.response_code,
+        originator: payment)
+      expect(result).to be_an_instance_of(ActiveMerchant::Billing::Response)
+      expect(result).to be_success
+    end
+
+    it 'returns an active merchant billing failure response' do
+      allow(SolidusPayTomorrow::Client::VoidService).to receive(:call).and_raise(StandardError)
+      result = described_class.new.void(payment.response_code,
+        originator: payment)
+      expect(result).to be_an_instance_of(ActiveMerchant::Billing::Response)
+      expect(result).not_to be_success
```

Comment:
The 2 examples looks very strange.
Do you mind creating a different context for them?